common/msg.c

```diff
@@ -299,15 +299,17 @@ bool mp_msg_has_status_line(struct mpv_global *global)
 
 static void set_term_color(void *talloc_ctx, bstr *text, int c)
 {
-    return c == -1 ? bstr_xappend(talloc_ctx, text, bstr0("\033[0m"))
-                   : bstr_xappend_asprintf(talloc_ctx, text,
-                                           "\033[%d;3%dm", c >> 3, c & 7);
+    if (c == -1)
+        bstr_xappend(talloc_ctx, text, bstr0("\033[0m"));
+    else
+        bstr_xappend_asprintf(talloc_ctx, text,
+                              "\033[%d;3%dm", c >> 3, c & 7);
```

Comment:
mpv style guidelines require bracket for such cases. Also could early return instead of else